javascript.js

```diff
@@ -11,4 +11,288 @@ document.getElementById('open-menu').onclick = openNav;
+  live: '#',
+  source: '#',
+};
+
+const project4 = {
+  title: 'Profesional Art Printing Data',
+  paragraph: `A daily selection of privately personalized reads;
+   no accounts or sign-ups required. has been the industry's standard`,
+  tecnologies: ['HTML', 'Bootstrap', 'Ruby'],
+  image: 'url(src/bg-project.svg)',
+  imagepopup: 'src/project-popup.svg',
+  popupdescription: `Lorem Ipsum is simply dummy text of the printing and 
+  typesetting industry. Lorem Ipsum has been the industry's standard dummy 
+  text ever since the 1500s, when an unknown printer took a galley of type 
+  and scrambled it to make a type specimen book. It has survived not only 
+  five centuries, but also the leap into electronic typesetting, remaining essent`,
+  live: '#',
+  source: '#',
+};
+
+const project5 = {
+  title: 'Profesional Art Printing Data',
+  paragraph: `A daily selection of privately personalized reads;
+   no accounts or sign-ups required. has been the industry's standard`,
+  tecnologies: ['HTML', 'Bootstrap', 'Ruby'],
+  image: 'url(src/bg-project.svg)',
+  imagepopup: 'src/project-popup.svg',
+  popupdescription: `Lorem Ipsum is simply dummy text of the printing and 
+  typesetting industry. Lorem Ipsum has been the industry's standard dummy 
+  text ever since the 1500s, when an unknown printer took a galley of type 
+  and scrambled it to make a type specimen book. It has survived not only 
+  five centuries, but also the leap into electronic typesetting, remaining essent`,
+  live: '#',
+  source: '#',
+};
+
+const recentWorks = {
+  image: 'src/Img%20Placeholder.svg',
+  imagepopup: 'src/project-popup.svg',
+  title: 'Multi-Post Stories',
+  paragraph: `A daily selection of privately personalized reads; 
+  no accounts or sing-ups required. has been the industry's standard 
+  dummy text ever since the 1500s, when an unknown printer took a standerd dummy text.`,
+  popupdescription: `Lorem Ipsum is simply dummy text of the printing and typesetting 
+  industry. Lorem Ipsum has been the industry's standard dummy text ever since the 
+  1500s, when an unknown printer took a galley of type and scrambled it 1960s with the
+   releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
+    Ipsum has been the industry's standard dummy text ever since the 1500s, when an 
+    unknown printer took a galley of type and scrambled it 1960s with the releax map 
+    lapora verita.`,
+  tecnologies: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+  live: '#',
+  source: '#',
+};
+
+const projectsArray = [
+  project0,
+  project1,
+  project2,
+  project3,
+  project4,
+  project5,
+  recentWorks,
+];
+
+function popup(num) {
+  document.getElementById('popup').innerHTML = `
+  <div class="darkpage"></div>
+  <div class="popup-container" >
+    <button id="close-popup" class="close-pu" type="button">&times;</button>
+    <h3 class="works-title-2 title-pu">${projectsArray[num].title}</h3>
```

Comment:
- [ ] Hi, the HTML that is generated for the `innerHTML` is not DRY. you are continuously repeating HTML tags for different projects. please loop through the projects and create only the HTML that is required for that specific project. avoid repeating yourself in your code. this is very hard to maintain. 
- [ ] Imagine a scenario where you have many more entries in an array. they way you've written your code currently will make it very difficult to maintain. Please make this change and everywhere else where you are directly indexing an array with numbers rather than variables